backend/app/services/chat/trigger/core.py

```
@@ -1320,3 +1343,75 @@ async def _stream_with_bridge(
+            .first()
+        )
+
+        if not ghost_kind or not ghost_kind.json:
+            return {}
+
+        # Parse Ghost CRD to get mcpServers
+        ghost_crd = Ghost.model_validate(ghost_kind.json)
+        if not ghost_crd.spec or not ghost_crd.spec.mcpServers:
+            return {}
+
+        return ghost_crd.spec.mcpServers
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)